multiversx_sdk_cli/cli_shared.py

```diff
@@ -259,3 +262,20 @@ def check_if_sign_method_required(args: List[str], checked_method: str) -> bool:
             return False
 
     return True
+
+
+def prepare_forwarded_command_arguments(args: Any) -> List[str]:
+    arguments = copy.deepcopy(args)
+    args_dict: Dict[str, Any] = arguments.__dict__
+
+    # delete the function key because we don't need to pass it along
+    args_dict.pop("func")
```

Comment:
Tested it locally, turns out we can't have an argument called `--func`. It this will ever be needed it should be called `--function`. `args.func` it's reserved for the function that get's called in our case.